### PR TITLE
Remove crypt

### PR DESCRIPTION
**Pull request by izik1**
Not sure how I ended up with 2 PRs for 2 things tonight, but I did

### CODE REVIEW

File: src/digraph.c [hunk 1]
```diff
@@ -40,62 +40,62 @@ static digr_T digraphdefault[] =
 	 * ATARI digraphs
 	 */
     {{'C', ',', 128},  /* ~@ XX */
-     {'u', '"', 129},  /* � */
```
Comment (izik1) [inline src/digraph.c:43]: Well, that broke and I never noticed...

Comment (izik1) [reply src/digraph.c:43]: It's only comments though? Maybe we can put UTF-8 in its place?

Comment (bryphe) [reply src/digraph.c:43]: Ah, interesting. I think replacing with UTF-8 would be reasonable.

**Comment by bryphe** [comment]
Awesome, thanks @izik1 for this streamlining. Looks good to me!

We can look at fixing the `digraph.c` in a separate PR.

**Pull request merged**